include/wx/string.h

```diff
@@ -241,7 +241,7 @@ class wxCStrData
 
 private:
     // the wxString this object was returned for
-    const wxString *m_str;
+    const wxString *m_str{nullptr};
```

Comment:
I'd rather avoid doing this as this gives the impression that this pointer can be null, which is not the case, it must always be valid.